### PR TITLE
Upgrade past security vulns

### LINKED ISSUE #4: Upgrade request->cryptiles, please
image

```
$ yarn why cryptiles
yarn why v1.17.0
[1/4] 🤔  Why do we have the module "cryptiles"...?
[2/4] 🚚  Initialising dependency graph...
[3/4] 🔍  Finding dependency...
[4/4] 🚡  Calculating file sizes...
=> Found "cryptiles@2.0.5"
info Reasons this module exists
   - "i18next-node-locize-backend#request#hawk" depends on it
   - Hoisted from "i18next-node-locize-backend#request#hawk#cryptiles"
info Disk size without dependencies: "36KB"
info Disk size with unique dependencies: "148KB"
info Disk size with transitive dependencies: "360KB"
info Number of shared dependencies: 2
✨  Done in 0.63s.
```

In other news; I did a review of the most up-to-date isomorphic (universal?) fetch façades; and [`cross-fetch`](https://www.npmjs.com/package/cross-fetch) came out on top. Perhaps use that instead of `request`?

### PR DESCRIPTION
**Pull request by haf**
Fixes #4 #5 #7

### CODE REVIEW

**Comment by haf** [comment]
Is this interesting?

File: .gitignore [hunk 1]
```diff
@@ -6,14 +6,12 @@
 npm-debug.log
 npm-debug.log.*
 *.dat
-package-lock.json
```
Comment (adrai) [inline .gitignore]: I would not commit package-lock.json, rest ok.

Comment (haf) [reply .gitignore]: How do you make sure you get repeatable builds without package-lock?

Comment (adrai) [reply .gitignore]: With tests.
Just my opinion and personal advice: _I would not commit package-lock.json on modules (only if the module is "big enough" and there is a bigger amount of collaborators), but only on projects._

**Comment by haf** [comment]
Fine, done!

**Comment by jamuhl** [comment]
published in i18next-node-locize-backend@2.4.3

**Pull request merged**